recipes/cuda-nvml-dev/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "cuda-nvml-dev" %}
+{% set version = "12.0.76" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvml_dev/{{ platform }}/cuda_nvml_dev-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 6780e554784bef73050144afe39668e26923d2533c268460c348e0e380d5d048  # [linux64]
+  sha256: 319b96999d144192094d254f0607debb14118ce206ef7dfeba3d13da1cc805ec  # [ppc64le]
+  sha256: 4aee09e48ecd30b2dcaa9de4eb8af2fcdf36ed040eac50c14f3345a12410b6ec  # [aarch64]
+  sha256: abc366280d0aa1ca047323a3e316270b80ffabb649021184647cd16b44c60a71  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+  host:
+    - cuda-version {{ cuda_version }}
+  run:
+    - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+  run_constrained:
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+
+test:
+  commands:
+    - test -f $PREFIX/targets/{{ target_name }}/lib/stubs/libnvidia-ml.so  # [linux]
+    - test -f $PREFIX/targets/{{ target_name }}/include/nvml.h             # [linux]
+    - if not exist %LIBRARY_LIB%\nvml.lib exit 1  # [win]
+    - if not exist %LIBRARY_INC%\nvml.h exit 1    # [win]
+    - if not exist %PREFIX%\nvml\example\example.c exit 1    # [win]
```

Comment:
Should we place this in a doc package?

Reply:
We could. Another question would be where we should install it on Windows? On Linux it will be here: `${PREFIX}/${targetsDir}/${PKG_NAME}`, but I don't think we decided on a similar place for Windows.

Reply:
We decided not to ship Fortran bindings for libcublas on Windows partly for this reason, I think. I don't have an opinion here. I think it's acceptable to ship the example somewhere in this package without splitting since it's only about 20 kB (not like the 10 MB of PDF files in cuda-cupti).

Reply:
I vote for not shipping this

Reply:
I also vote to not ship, at least for now.